composer.json

```
@@ -68,7 +68,7 @@
 		"oat-sa/extension-tao-delivery-rdf" : ">=14.0.0",
 		"oat-sa/extension-tao-lti" : ">=15.11.0",
 		"oat-sa/extension-tao-outcomeui" : ">=10.0.0",
-		"oat-sa/extension-tao-testqti" : ">=42.2.0",
+		"oat-sa/extension-tao-testqti" : "dev-feat/TR-5852/disable-multiple-delivery-launch as >=42.2.0",
```

- [ ] We need to fix it in order for the installation to work

```suggestion
		"oat-sa/extension-tao-testqti" : "dev-feat/TR-5852/disable-multiple-delivery-launch as 42.2.0",
```